### ISSUE TITLE
Unable to update module due to no cache introduced in #528

### ISSUE THREAD

**Issue by william00179** [npalm/terraform-aws-gitlab-runner#541]
Hi, 

Since #528 has been merged, we have not been able to deploy the module.

We have standalone runners deployed alongside some infrastructure so we can distribute jobs to the various runners and these runners are the only things authorized to communicate with the services.

Thus there is no need for us to employ a cache. Previously this configuration worked for us;

```
cache_bucket = {
    "bucket": "",
    "create": false,
    "policy": ""
  }
  ```

On apply the following error is now raised

```
Error: Error attaching policy  to IAM Role redacted: InvalidParameter: 1 validation error(s) found.
│ - minimum field size of 20, AttachRolePolicyInput.PolicyArn.
```

**Comment by kayman-mk** [comment]
Thanks for the report. This is addressed #540 and is waiting for the merge.

**Comment by npalm** [comment]
closing since #540 is merged and released (5.3.0)
